mail_footer_notified_partners/models/mail_followers.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     This file is part of mail_footer_notified_partners,
+#     an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     mail_footer_notified_partners is free software:
+#     you can redistribute it and/or modify it under the terms of the GNU
+#     Affero General Public License as published by the Free Software
+#     Foundation,either version 3 of the License, or (at your option) any
+#     later version.
+#
+#     mail_footer_notified_partners is distributed
+#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with mail_footer_notified_partners.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, api
+from openerp import tools
+from openerp.tools.translate import _
+
+ADDITIONAL_FOOTER = _('This message was also sent to: ')
```

Comment:
this will give you the wrong translation because the translation is not executed in the user's context but evaluated on load. Better put this into the code